src/web/tools/PointerEventManager.ts

```diff
@@ -22,200 +32,234 @@ export default class PointerEventManager extends EventManager<HTMLElement> {
     this.mouseButtonsMapper.set(2, MouseButton.RIGHT);
     this.mouseButtonsMapper.set(3, MouseButton.BUTTON_4);
     this.mouseButtonsMapper.set(4, MouseButton.BUTTON_5);
+
+    this.boundPointerDownCallback = this.pointerDownCallback.bind(this);
+    this.boundPointerUpCallback = this.pointerUpCallback.bind(this);
+    this.boundPointerMoveCallback = this.pointerMoveCallback.bind(this);
+    this.boundPointerCancelCallback = this.pointerCancelCallback.bind(this);
+    this.boundPointerEnterCallback = this.pointerEnterCallback.bind(this);
+    this.boundPointerLeaveCallback = this.pointerLeaveCallback.bind(this);
+    this.boundLostPointerCaptureCallback =
+      this.lostPointerCaptureCallback.bind(this);
```

Comment:
Cannot arrow functions be used instead of binding (same in `TouchEventManager`)?

Reply:
You're right! For some reason I simply forgot that it is possible to define arrow function directly inside class 🤦

Reply:
Done in [dd9f77e](https://github.com/software-mansion/react-native-gesture-handler/pull/2815/commits/dd9f77ed14789df64dff1b2b9087d45e370d760b).